create/using-manual-approval.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-manual-approval"]
+= Using manual approval in {pipelines-shortname}
+:context: using-manual-approval
+
+toc::[]
+
+You can specify a manual approval task in a pipeline. When the pipeline reaches this task, it pauses and awaits approval from one or several {OCP} users. If any of the users chooses to rejects the task instead of approving it, the pipeline fails.
+
+The manual approval gate controller provides this functionality.
```

Comment:
This sentence is dependant on the previous paragraph, so how about add it to the previous par?